tests/helpers.py

```diff
@@ -419,12 +419,12 @@ def create_cephfilesystem():
     POD = pod.get_all_pods(
         namespace=defaults.ROOK_CLUSTER_NAMESPACE
     )
-    for pod_names in POD:
-        if 'rook-ceph-mds' in pod_names.labels.values():
-            assert pod_names.ocp.wait_for_resource(
-                condition=constants.STATUS_RUNNING,
-                selector='app=rook-ceph-mds'
-            )
+    assert POD[0].ocp.wait_for_resource(
+        condition=constants.STATUS_RUNNING,
+        selector='app=rook-ceph-mds',
+        timeout=120
```

Comment:
create a global for 120

Reply:
not really but hard-coded values are not good practice

Reply:
generally local values are defined in the same class file instead of other modules, we should do the same here.

Reply:
I don't agree that hard-coded values are bad practice 100% of the time, and I think that a timeout is actually a really good use-case for one. Since different operations will have different needs for a timeout, I don't see the value added by moving it to a global variable. Even if multiple operations need 120 for their timeout, that might not always be the case, and they might not always need the same value.

In a use-case where you DO want to change values across the board, I totally agree that hard-coding the values is bad practice and we would want them to use a variable instead. That being said, I don't think this timeout value falls into that bucket.